libraries/tools/kotlin-stdlib-gen/src/templates/Ranges.kt

```diff
@@ -76,6 +76,7 @@ object RangeOps : TemplateGroupBase() {
             return TProgression.fromClosedRange(first, last, if (this.step > 0) step else -step)
             """
         }
+        sample("samples.ranges.Ranges.step${primitive!!.stepType}")
```

Comment:
Using `stepType` is incorrect here because it can be only `Int` or `Long`

Reply:
yep. fixed. sorry about that.